Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -126,6 +127,10 @@ function Get-MsDeployCmdArgs
         $msDeployCmdArgs += [String]::Format(' -skip:Directory="\\App_Data"')
     }
 
+    if( -not [String]::IsNullOrEmpty($setParametersFile)){
```

Comment:
This may never be NullOrEmpty because of the default behavior of filePath type inputs.
FilePathType inputs default to SYSTEM_DEFAULTWORKINGDIRECTORY. So, we should check the value and reset it to "".

Ref: $publishProfile input in [https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/SqlServerDacpacDeployment/DeployToSqlServer.ps1](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/SqlServerDacpacDeployment/DeployToSqlServer.ps1)